### PR TITLE
Issue 74b: IABookreader, please cooperate with old friend PDF?

### PR DESCRIPTION
**Pull request by DiegoPino**
See #74 

# What is this?

Some JS magic to deal with missing/partial/missunderstood Manifests.  It does the following
- If service level is > 0 (means we have an info.json), append the get arguments to it (?page=1)
- If width/height is present but level == 0 (like static image or one we want o set because we are like that) then use the Canvas width/height ratio to calculate the missing one. (either height or width)
- If service level == 0 don't even bother trying to figure out an .info.json and in the worst possible condition just use the generic 3/4 ration and console it out so the webmaster/admin/can have nightmares and fix its Twig Template.

Gosh! @giancarlobi i know you out but someone needs to see this even if in 2 days. @alliomeria i need to introduce you IIIF Manifests sooner than later.

In conclusion. This allows Bookreader to display individual pages for PDFs without any issue, even if we don't want to fake the size

### CODE REVIEW

**Comment by DiegoPino** [comment]
Merging folks. No time to loose. Also going to bed. Have been up more than it's legal 😴
